girder/api/rest.py

```diff
@@ -582,15 +582,27 @@ def _handleValidationException(e):
     return val
 
 
+def disableAuditLog(fun):
```

Comment:
I think I missed this in the original PR for the audit logs plugin. I thought everything was contained within a plugin, but it appears that some of the audit logging code is in Girder core?

Reply:
The emission of log entries comes from core (since that's what we are interested in logging). The handling of those events is in the plugin.